lib/helpers/UiBaseLocators.ts

```diff
@@ -108,9 +132,10 @@ export class UiBaseLocators {
     await this.page.waitForTimeout(1000);
     await this.modalCaretBtn.click({force: true});
     await this.page.getByLabel(dictionaryName).click();
-    await this.submitBtn.click();
+    await this.chooseDictionaryBtn.click();
   }
 
+  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
```

Comment:
```suggestion
  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
```

Reply:
I will implement it later to avoid the conflict